pom.xml

```diff
@@ -224,7 +224,7 @@
     <spring.boot.version>2.7.11</spring.boot.version>
     <spring-netflix.version>3.1.7</spring-netflix.version>
     <spring-cloud.version>2021.0.8</spring-cloud.version>
-    <spring.cloud.nacos.version>2.2.9.RELEASE</spring.cloud.nacos.version>
+    <spring.cloud.nacos.version>2021.0.5.0</spring.cloud.nacos.version>
```

Comment:
this should rename to spring-cloud-alibaba.version,   and the version can upgrade to 2021.0.6.0